VK_V1.0/VK_V1.0/LoginViewController.swift

```diff
@@ -0,0 +1,136 @@
+// LoginViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Экран входа
+final class LoginViewController: UIViewController {
+    // MARK: - Private Enum
+
+    private enum Constants {
+        static let segueIdentifier = "loginVC"
+        static let loginText = "a"
+        static let passwordText = "1"
+        static let alertTitleText = "Ошибка"
+        static let alertMessageText = "Логин и/или пароль неверны."
+        static let actionOkText = "OK"
+    }
+
+    // MARK: - Private Visual Components
+
+    @IBOutlet private var passwordTextView: UITextField!
+    @IBOutlet private var loginTextView: UITextField!
+    @IBOutlet private var scrollView: UIScrollView!
+
+    // MARK: - LifeCycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        addObserver()
```

Comment:
```suggestion
        addObserver()
```
в уроке мы добавляли обсерверы во viewWillAppear, не знаю насколько принципиально но можно сделать по уроку